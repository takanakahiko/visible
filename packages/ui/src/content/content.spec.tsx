import React from 'react';
import { render } from '@testing-library/react';
import { Content } from '.';

describe('Content', () => {
  it('matches snapshot', () => {
    const { container } = render(<Content />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
