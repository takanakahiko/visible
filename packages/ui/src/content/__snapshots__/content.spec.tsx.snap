// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content matches snapshot 1`] = `
.c0 {
  box-sizing: border-box;
  width: 1080px;
  margin: 24px auto;
  padding: 12px 18px;
  border-radius: 12px;
  background-color: white;
  box-shadow: 0 0 24px rgba(0,0,0,0.05);
}

<main
  class="c0"
/>
`;
